modules/ROOT/pages/configure-cidr-range.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Pod CIDR Range for Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To configure the pod Classless Inter-Domain Routing (CIDR) range on an existing Runtime Fabric cluster:
+
+. Create a resource definition using a new `podCIDR` value. In the following example, the yaml file is named `pod-cidr.yaml` and `x.x.x.x/16` represents the new pod CIDR range:
++
+```
+kind: clusterconfiguration
+version: v1
+spec:
+  global:
+    podCIDR: x.x.x.x/16
+```
+. Run the following gravity command:
++
+```
+sudo gravity resource create pod-cidr.yaml
```

Comment:
missing `--confirm` at the end of the command.

Reply:
Added - thank you!